cmake/sysbuild/provision_hex.cmake

```diff
@@ -125,6 +125,7 @@ function(provision application prefix_name)
 
   add_custom_target(
     ${prefix_name}provision_target
+    ALL
```

Comment:
why is `ALL` needed here ?

The merged_hex target already depends on `merged.hex` file creation which again depends on the `<name>_provision.hex` file creation.

https://github.com/nrfconnect/sdk-nrf/blob/41c2ad2b39a8408059432f0a54495763f7db9723/cmake/sysbuild/partition_manager.cmake#L272-L282
https://github.com/nrfconnect/sdk-nrf/blob/41c2ad2b39a8408059432f0a54495763f7db9723/cmake/sysbuild/partition_manager.cmake#L253

Reply:
Fixed